src/renderers/webxr/WebXRManager.js

```diff
@@ -537,6 +550,15 @@ class WebXRManager extends EventDispatcher {
 				_currentDepthNear = cameraXR.near;
 				_currentDepthFar = cameraXR.far;
 
+				cameraL.near = depthSensing.depthNear; // ?
+				cameraL.far = depthSensing.depthFar; // ?
+				cameraR.near = depthSensing.depthNear; // ?
+				cameraR.far = depthSensing.depthFar; // ?
+
+				cameraL.updateProjectionMatrix();
+				cameraR.updateProjectionMatrix();
+				camera.updateProjectionMatrix();
+
```

Comment:
@cabanier Do you mind explaining these lines? If `depth-sensing` is not available we're setting these to 0, right?

Reply:
no, we should leave these to what the author set them or the WebXR default.

Reply:
These values come from the author and browser uses them to calculate the matrices

Reply:
So the code looks good to you? If not, feel free to clean up the PR.

Reply:
If I submitted this, it is wrong. It should be using _currentDepthNear/_currentDepthFar

Reply:
Ok, please change it to the correct values when you get a chance.

Reply:
done